File: src/main/java/ru/translater/format/model/Page.java
package ru.translater.format.model;

import com.fasterxml.jackson.databind.JsonNode;
import lombok.Data;
import ru.translater.common.model.Meta;

import java.util.ArrayList;
import java.util.List;

@Data
public class Page {
    private String sourceUrl;
    private String uniqueId;
    private String generatedDate;
    //TODO move to payload
    private String pictureUrl;
    private List<Meta> metas = new ArrayList<>();

    private JsonNode payload;
}

File: src/main/kotlin/com/translater/format/model/Page.kt
package com.translater.format.model

import com.fasterxml.jackson.databind.JsonNode
import com.translater.common.model.Meta

data class Page(
    var sourceUrl: String? = null,
    var uniqueId: String? = null,
    var generatedDate: String? = null,
    //TODO move to payload
    var pictureUrl: String? = null,
    var metas: MutableList<Meta> = mutableListOf(),
    var payload: JsonNode? = null
)